### ISSUE TITLE
Cloud Config: VM still running after experiment failure.

### ISSUE THREAD

**Issue by mvanotti** [google/fuzzbench#1807]
I tried to launch an experiment in my own cloud project and something failed, however, the VM was created and seems to be running.

I would like to be able to cancel and delete all resources that were used in case of failure.

**Comment by Alan32Liu** [comment]
Yep, you can stop an experiment and all its instances with [`experiment/stop_experiment.py`](https://github.com/google/fuzzbench/blob/master/experiment/stop_experiment.py).
